src/rpc/mining.cpp

```diff
@@ -247,7 +247,7 @@ static UniValue prioritisetransaction(const JSONRPCRequest& request)
 
     LOCK(cs_main);
 
-    uint256 hash = ParseHashStr(request.params[0].get_str(), "txid");
+    uint256 hash = ParseHashV(request.params[0], "txid");
```

Comment:
Note this trades `ParseHashStr`'s `std::runtime_error`-based encoding-only test for `ParseHashV`'s `JSONRPCError`-based length and encoding tests.